modules/installation-osp-enabling-swift.adoc

```diff
@@ -7,14 +7,20 @@
 [id="installation-osp-enabling-swift_{context}"]
 = Enabling Swift on OpenStack
 
-{product-title} on {rh-openstack-first} uses https://docs.openstack.org/security-guide/object-storage.html[OpenStack Object Storage (Swift)] to store and serve user configuration files.
+Swift is operated by a user account with the `swiftoperator` role. Add the role to an account before you run the installation program.
 
-Swift is operated by a user account with the `swiftoperator` role.
+[IMPORTANT]
+====
+If link:https://access.redhat.com/documentation/en-us/red_hat_openstack_platform/16.0/html-single/storage_guide/index#ch-manage-containers[OpenStack Object Storage (Swift)] is available, {product-title} on {rh-openstack-first} uses it as the image registry storage. If it is unavailable, the installation program relies on the OpenStack Block Storage service (Cinder).
+
+If Swift is present and you want to use it, you must enable access to it. If it is not present, or if you do not want to use it, skip this section.
+====
 
 .Prerequisites
 
-* A {rh-openstack} administrator account on the target environment
-* On Ceph RGW, https://docs.ceph.com/docs/master/radosgw/config-ref/#swift-settings[the `account in url` option must be enabled]
+* You have a {rh-openstack} administrator account on the target environment
+* The Swift service is installed
+* On link:https://access.redhat.com/documentation/en-us/red_hat_openstack_platform/16.0/html-single/deploying_an_overcloud_with_containerized_red_hat_ceph/index#ceph-rgw[Ceph RGW], the `account in url` option is enabled
```

Comment:
Do you need periods at the end of each sentence in the list?

Reply:
ISG says yes, so yes.